src/mantle/testkit/concerns/trait-installs-wordpress.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Installs_WordPress trait file
+ *
+ * @package Mantle
+ */
+
+namespace Mantle\Testkit\Concerns;
+
+/**
+ * Concern for handling WordPress installation outside of the bootstrap.
+ */
+trait Installs_WordPress {
+	/**
+	 * Manages installing WordPress.
+	 *
+	 * This is useful for installing WordPress in integration tests, while leaving the bootstrap
+	 * free of this overhead during unit tests. To perform actions after WordPress is installed, you
+	 * simply need to define a function `mantle_after_wordpress_install` in your bootstrap file.
+	 */
+	public static function installs_wordpress_set_up_before_class(): void {
+		static $installed = false;
+
+		if ( true === $installed ) {
+			return;
+		}
+
+		$callback = function_exists( 'mantle_after_wordpress_install' ) ? 'mantle_after_wordpress_install' : null;
+
+		\Mantle\Testing\install( $callback );
```

Comment:
I wonder if this should be a filter instead of a pre-defined function? /shrug

Reply:
I thought about that, too, but I don't think at this point we'd have access to filters, unless there is a way to do that built into Mantle. Another option could be to look for the existence of a class method, but that would require extending `Integration_Test_Case` which pulls it away from how `Unit_Test_Case` works, which seemed not ideal.

Reply:
eh, not a blocker at all. i think it's good as is!